src/main/java/survey/Question.java

```diff
@@ -0,0 +1,36 @@
+package survey;
+
+public abstract class Question {
```

Comment:
Should this be a interface? So other question types can implement from it.

Reply:
I am not sure because I think it needs to fit into the database and I am not sure if it works for inheritance. We need to figure that out.